Source/WebKit/Configurations/BaseExtension.xcconfig

```diff
@@ -37,6 +38,7 @@ SUPPORTS_MAC_DESIGNED_FOR_IPHONE_IPAD = NO;
 TARGETED_DEVICE_FAMILY = 1,2,7;
 EXCLUDED_SOURCE_FILE_NAMES = *;
 EXCLUDED_SOURCE_FILE_NAMES[sdk=iphoneos17.4.internal] = ;
+EXCLUDED_SOURCE_FILE_NAMES[sdk=iphonesimulator17.4.internal] = ;
```

Comment:
Ditto. Here, it's may be to do something like:
```suggestion
EXCLUDED_SOURCE_FILE_NAMES = $(EXCLUDED_SOURCE_FILE_NAMES_$(USE_INTERNAL_SDK));;
EXCLUDED_SOURCE_FILE_NAMES_ = *;
EXCLUDED_SOURCE_FILE_NAMES_YES = *;
EXCLUDED_SOURCE_FILE_NAMES_YES[sdk=iphone*] = ;
EXCLUDED_SOURCE_FILE_NAMES_YES[sdk=iphone*17.0*] = *;
EXCLUDED_SOURCE_FILE_NAMES_YES[sdk=iphone*17.1*] = *;
EXCLUDED_SOURCE_FILE_NAMES_YES[sdk=iphone*17.2*] = *;
EXCLUDED_SOURCE_FILE_NAMES_YES[sdk=iphone*17.3*] = *;
```

Reply:
Ah, good catch, will fix!